monsteer/plugin/spikeReport.cpp

```diff
@@ -49,19 +50,26 @@ namespace
 const uint32_t INTERNAL_TIMEOUT = 500;
 }
 
+URI toHostAndPort( const URI& uri )
+{
+    URI out;
+    out.setHost( uri.getHost( ));
+    out.setPort( uri.getPort( ));
+    return out;
+}
+
 SpikeReport::SpikeReport( const brion::SpikeReportInitData& pluginData )
-    : _uri( pluginData.getURI( ))
+    : _uri( toHostAndPort( pluginData.getURI( )))
```

Comment:
Yes another thing I wanted to do, but that work because it doesn't remove the monsteer:// scheme. Which is why I wanted to add something like the following to Servus:
static URI fromHostAndPort( const std::string& uri );
std::string toHostAndPort() const;
Or even introduce a proper HostPort class...